- Implemented a new button 'Select S.O' in the app - The button provi…

SufyanSadiqOffice opened the pull request:
…des a list of Sales Orders for selection - Users can now create Sales Invoices for the selected Sales Order

yrestom committed:
Merge pull request #338 from yrestom/develop

Version: 6.0.3

yrestom committed:
Merge pull request #345 from yrestom/develop

Version: 6.0.4

yrestom committed:
Merge pull request #355 from yrestom/develop

Version: 6.1.0

yrestom committed:
Merge pull request #357 from yrestom/develop

Version: 6.1.1

yrestom committed:
Merge pull request #358 from yrestom/develop

Version: 6.1.2

yrestom committed:
Merge pull request #371 from yrestom/develop

Version: 6.1.3

SufyanSadiqOffice committed:
- Implemented a new button 'Select S.O' in the app - The button provides a list of Sales Orders for selection - Users can now create Sales Invoices for the selected Sales Order

sowaan commented:
closes #416

yrestom commented:
@SufyanSadiqOffice 
It is nice feature.
can you please clean the PR from unnecessary changes?

SufyanSadiqOffice committed:
Revert "- Implemented a new button 'Select S.O' in the app - The button provides a list of Sales Orders for selection - Users can now create Sales Invoices for the selected Sales Order"

This reverts commit 4d23573d1f87c80235c1943d2c2b7440aa7a3b9c.

SufyanSadiqOffice committed:
Feat : implemented a new Select S.O Button.

The button provides a list of Sales Orders for selection - Users can now create Sales Invoices for the selected Sales Order.

CloudERPng commented:
IF this is merged how do we us it? I see the option in Pos Profile, but not able to see it at the front end

CloudERPng commented:
![image](https://github.com/yrestom/POS-Awesome/assets/38726533/bfa61282-49b0-42ce-8133-f88498167571)

CloudERPng commented:
![image](https://github.com/yrestom/POS-Awesome/assets/38726533/b869a5af-cdf5-449f-8408-cdb4f1fdc850)
